frontend/src/screens/Intro.tsx

```diff
@@ -87,6 +88,14 @@ export function Intro() {
             </div>
           </div>
         </CarouselItem>
+        <CarouselItem>
```

Comment:
this should only show if the user actually has funds

Reply:
Ah okay 👍

Reply:
How can we know if the user has hosted balance in the intro itself?

Reply:
Sorry, yeah I was wrong. It's probably fine as-is